packages/checkout/widgets-lib/src/resources/text/textConfig.ts

```diff
@@ -67,6 +67,10 @@ export const text = {
       body: ['You can try again or contact', 'support', 'for help.'],
       actionText: 'Try again',
     },
+    [SharedViews.SERVICE_UNAVAILABLE_ERROR_VIEW]: {
+      heading: 'Service not available in your region',
+      body: 'We’re sorry we cannot provide this service in your region.',
```

Comment:
@sharonsheah @deepti-imx 
Can we make this copy more specific?
For instance, instead of saying 'Service not available', we could at least say which service is not available. "Token swaps are not available in you region"

Also, do we have product direction to say why they are not available in the region, do we specify which regions are supported anywhere?

Reply:
> Can we make this copy more specific?
The error view is generic and the copy is provided by legal team (prettified by Cass) but can check with Cass again on the copy.

> Also, do we have product direction to say why they are not available in the region, do we specify which regions are supported anywhere?

We specify the geo-blocked regions in developer documentation.
For end-users, we don't have any documentation yet. It is something we have discussed to support the end-users but not prioritised the docs yet.

Reply:
Bringing this conversation to slack to include Harris and Cass https://imtbl.slack.com/archives/C0414KQ5162/p1696904344641679